### PR TITLE
Genarate data button

### PR DESCRIPTION
**Pull request by Chiaroks**

### CODE REVIEW

File: src/DataBaseTree/ViewModels/MetadataTree/TreeViewModel.cs [hunk 1]
```diff
@@ -7,17 +7,29 @@
...
     public class TreeViewModel : ViewModelBase
     {
         private ICommand _setSelectedItemCommand;
         private ICommand _copyItem;
+        private ICommand _showDataGenerationConfigWindow;
 
         private TreeViewItemViewModelBase _selectedItem;
 
         public ObservableCollection<TreeViewItemViewModelBase> RootItems { get; }
+        public ICommand ShowDataGenerationConfigWindow => _showDataGenerationConfigWindow ?? (_showDataGenerationConfigWindow = new RelayCommand(s => ExecuteShowDataGenerationConfigWindow()));
```
Comment (meCtd) [inline src/DataBaseTree/ViewModels/MetadataTree/TreeViewModel.cs]: move to other commands please (ordering) and blank lines before and after member

File: src/DataBaseTree/ViewModels/Windows/DataGanarationConfigurationViewModel.cs [hunk 2]
```diff
@@ -0,0 +1,16 @@
+﻿using DBManager.Application.ViewModels.General;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DBManager.Application.ViewModels.Windows
+{
+    class DataGanarationConfigurationViewModel : WindowViewModelBase
```
Comment (meCtd) [inline src/DataBaseTree/ViewModels/Windows/DataGanarationConfigurationViewModel.cs]: Spelling

File: src/DataBaseTree/Views/Templates/DataGenerationConfigWindowTemplate.xaml [hunk 3]
```diff
@@ -0,0 +1,11 @@
+﻿<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+                    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+                    xmlns:local="clr-namespace:DBManager.Application.Views.Templates"
+                    xmlns:windows="clr-namespace:DBManager.Application.ViewModels.Windows">
+
+    <DataTemplate DataType="{x:Type windows:DataGanarationConfigurationViewModel}">
```
Comment (meCtd) [inline src/DataBaseTree/Views/Templates/DataGenerationConfigWindowTemplate.xaml]: empty window :D ??

File: src/DataBaseTree/Views/Templates/MetadataTreeTemplates.xaml [hunk 4]
```diff
@@ -9,7 +9,7 @@
                     xmlns:i="http://schemas.microsoft.com/expression/2010/interactivity"
                     xmlns:metadataTree="clr-namespace:DBManager.Application.ViewModels.MetadataTree">
 
-	<convertors:SelectTop100RowsButtonVisibilityConverter x:Key="selectTop100Converter"/>
+	<convertors:TreeContextMenuButtonVisibilityConverter x:Key="selectTop100Converter"/>
```
Comment (meCtd) [inline src/DataBaseTree/Views/Templates/MetadataTreeTemplates.xaml]: rename convertor's resource key

File: src/DataBaseTree/Views/Templates/MetadataTreeTemplates.xaml [hunk 5]
```diff
@@ -129,8 +129,13 @@
 					<MenuItem Header="Select TOP 100 rows"
                               Command="{Binding Data.Scripts.SelectTopRowsCommand,Source={StaticResource Proxy}}"
                               CommandParameter="{Binding}"
-							  Visibility="{Binding Path=., Converter={StaticResource selectTop100Converter}}"
+							  Visibility="{Binding Path=., Converter={StaticResource selectTop100Converter},ConverterParameter={x:Static tree:MetadataType.DataObject }}"
```
Comment (meCtd) [inline src/DataBaseTree/Views/Templates/MetadataTreeTemplates.xaml]: path is unneded

File: src/DataBaseTree/Views/Templates/MetadataTreeTemplates.xaml [hunk 6]
```diff
@@ -129,8 +129,13 @@
...
                               Icon="{StaticResource NewScriptIcon}" />
+                    <Separator />
+                    <MenuItem Header="Genarate data"
+							  Visibility="{Binding Path=., Converter={StaticResource selectTop100Converter},ConverterParameter={x:Static tree:MetadataType.Table }}"
```
Comment (meCtd) [inline src/DataBaseTree/Views/Templates/MetadataTreeTemplates.xaml]: path unndeded, formatting

File: src/DataBaseTree/Views/Windows/ContentWindow.xaml [hunk 7]
```diff
@@ -22,7 +22,12 @@
         Title="{Binding Header}">
 
     <Window.Resources>
-        <utils:SharedResourceDictionary Source="pack://application:,,,/Views/Templates/ConenctionWindowTemplate.xaml" />
+        <ResourceDictionary>
+            <ResourceDictionary.MergedDictionaries>
+                <utils:SharedResourceDictionary Source="pack://application:,,,/Views/Templates/ConenctionWindowTemplate.xaml" />
+                <utils:SharedResourceDictionary Source="pack://application:,,,/Views/Templates/DataGenerationConfigWindowTemplate.xaml" />
```
Comment (meCtd) [inline src/DataBaseTree/Views/Windows/ContentWindow.xaml]: use full name - DataGenerationConfigurationWindowTemplate

File: src/DataBaseTree/Views/Templates/DataGenerationConfigWindowTemplate.xaml (hunk 3 above)
Comment (MrYandersen) [reply src/DataBaseTree/Views/Templates/DataGenerationConfigWindowTemplate.xaml]: I gave him this task. I will develop the contents of the window.

**Pull request merged**